### PR TITLE
Adding tracing component to the secret store allow list

### PR DESCRIPTION
**Pull request by mftb**

### CODE REVIEW

**Comment by manish-jangra** [comment]
@gbenhaim is that the only file to be edited?

File: components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml [hunk 1]
```diff
@@ -52,3 +52,4 @@ spec:
         - integration-service
         - ci-helper-app
         - mintmaker
+        - tracing
```
Comment (hugares) [inline components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml:55]: the namespace is `konflux-otel `, not `tracing`.
```suggestion
        - konflux-otel 
```

**Comment by hugares** [comment]
> @gbenhaim is that the only file to be edited?

yes it is, we are only missing the `konflux-otel` namespaces to be allowed to access the `appsre-stonesoup-vault` ClusterSecretStore

File: components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml (hunk 1 above)
Comment (mftb) [reply components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml:55]: TYVM, fixing

**Comment by mftb** [comment]
@hugares I've updated to use the proper namespace

**Review by hugares — APPROVED** [review]
/lgtm

**Comment by hugares** [comment]
flaky test and this change is not tested by e2e, merging

**Pull request merged**